loader/proto.go

```diff
@@ -88,6 +89,9 @@ type builder struct {
 	// Mostly these will be Gunk annotations. Import name will be
 	// mapped to its possible named import.
 	importsUsed map[string]string
+
+	// Holds existings declaration to avoid duplicate
+	existingDeclarations map[string]bool
```

Comment:
you could call this `existingDecls`, as that's a common abbreviation for the word.

Reply:
Indeed. I've renamed to `existingDecls` and amended my commit.